content/W2015/W15-blog-caleybrock.md

```diff
@@ -44,7 +44,7 @@ All of her experiences have taught her that creativity is not limited to the art
 
 Caley believes that there needs to be earlier exposure to computer science education in high school through mentorships.
 
-"There needs to be more one-on-one support and mentorships. When learning programming,  it’s frustrating to get an error and not know what to do. It doesn’t mean you’re stupid. You understand the concepts."
+"There needs to be more one-on-one support and mentorships. When learning programming, it's frustrating to get an error and not know what to do. It doesn't mean you're stupid. You understand the concepts."
```

Comment:
Since this is an extended quote, you don't need to end this paragraph in a quote.